src/pages/index.js

```diff
@@ -148,7 +251,41 @@ profileEditButton.addEventListener("click", () => {
   profileModal.open();
 });
 
+// Define the handleAvatarSubmit function
+// function handleAvatarSubmit({ inputData, form }) {
+//   const avatarUrl = inputData.avatar; // Get the URL from the form
+
+//   api
+//     .updateAvatar(avatarUrl)
+//     .then((data) => {
+//       // Update the avatar image src on the page
+//       avatarImage.src = data.avatar;
+//       form.reset();
+//     })
+//     .catch((error) => {
+//       console.error("Failed to update avatar:", error);
+//     });
+// }
+
+function handleAvatarSubmit({ inputData, form }) {
+  const avatarUrl = inputData.avatar; // Get the URL from the form
+
+  return api
+    .updateAvatar(avatarUrl)
+    .then((data) => {
+      userInfo.setUserInfo({ avatar: data.avatar });
+      form.reset();
```

Comment:
* The `avatar` save button should be disabled after submittion as well https://snipboard.io/9Qn5m6.jpg 


`avatarFormValidator.toggleButtonState();` should be here to disable the  avatar save button